README.md

```diff
@@ -182,18 +182,17 @@ and toolchains that are currently known to work (last updated 2022/01/24):
 ### Prerequisites
 
 - [Boost](https://www.boost.org/): program_options
-- [ROOT](https://root.cern/): RIO, Hist, Tree
```

Comment:
As much as I would like to, let's not _remove_ ROOT as a prerequisite. Rather, let's make it clear in the README that it is optional.

Reply:
You're correct that "optional" and "prerequisite" are technically a contradictio in terminis, but I would like to keep this here. You are also correct that CMake is a prerequisite.

We can rename this section to "Dependencies", which does not necessarily contradict the fact that ROOT is now optional.

You should also feel free to close this comment chain as I am really just arguing semantics at this point.